docs/snippets/svelte/button-implementation.js.mdx

```diff
@@ -0,0 +1,37 @@
+```html
+<!-- Button.svelte -->
+
+<script>
+  import { createEventDispatcher } from 'svelte';
+  /**
+   * Is this the principal call to action on the page?
+   */
+  export let primary = false;
+
+  /**
+   * What background color to use
+   */
+  export let backgroundColor;
+  /**
+   * How large should the button be?
+   */
+  export let size = 'medium';
+  /**
+   * Button contents
+   */
+  export let label = '';
+
+  let style = backgroundColor ? `background-color: ${backgroundColor}` : '';
```

Comment:
This line should be a reactive statement :

```suggestion
  $: style = backgroundColor ? `background-color: ${backgroundColor}` : '';
```

Reply:
Fixed with the latest commit